### PR TITLE
fix(currency): install google-cloud-storage v2.18.2 separately for python 3.11 and python 3.12

### PR DESCRIPTION
**Pull request by CagriYonca**

### CODE REVIEW

**Review by pvital — CHANGES_REQUESTED** [review]
Wait. You have just removed the job that runs GoogleCloud in the commit b7d66d1. 
Why you are re-introducing it again?

**Review by pvital — APPROVED** [review]
It looks good to me.

File: tests/requirements.txt [hunk 1]
```diff
@@ -13,7 +13,8 @@ flask>=2.3.2
 gevent>=1.4.0
 grpcio>=1.14.1
 google-cloud-pubsub>=2.0.0
-google-cloud-storage>=1.24.0
+google-cloud-storage>=1.24.0; python_version < "3.11" or python_version > "3.11"
```
Comment (GSVarsha) [inline tests/requirements.txt:16]:
```suggestion
google-cloud-storage>=1.24.0; python_version != "3.11"
```

**Review by GSVarsha — CHANGES_REQUESTED** [review]
Hi @CagriYonca, I don't quite understand the intention behind this PR.

[This](https://github.com/instana/python-sensor/pull/662/files#diff-27c5febcc9ce93e8e5341f263b7d68962733b8a2a86c583723d025a1db39d7a3R17) change doesn't reflect the title of the PR: `"run google-cloud-storage v2.18.2 separately for python 3.11"`.

The commit message: [combined requirement files into one file and updated pipelines](https://github.com/instana/python-sensor/pull/662/commits/133485d0432203a71b30afe3444c7fd06d4bf981) also refers to something else.

**Comment by CagriYonca** [comment]
> Hi @CagriYonca, I don't quite understand the intention behind this PR.
> 
> [This](https://github.com/instana/python-sensor/pull/662/files#diff-27c5febcc9ce93e8e5341f263b7d68962733b8a2a86c583723d025a1db39d7a3R17) change doesn't reflect the title of the PR: `"run google-cloud-storage v2.18.2 separately for python 3.11"`.
> 
> The commit message: [combined requirement files into one file and updated pipelines](https://github.com/instana/python-sensor/pull/662/commits/133485d0432203a71b30afe3444c7fd06d4bf981) also refers to something else.

It's just a fıx for fıx_requirements branch, I'll update the commit message right now

File: tests/requirements-googlecloud.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+google-cloud-storage>=2.15.0
+google-api-core>=2.15.0
```
Comment (GSVarsha) [inline tests/requirements-googlecloud.txt:2]: You can remove this since it is already required by `gcs`

```suggestion
```

**Review by GSVarsha — CHANGES_REQUESTED** [review]
Please add an entry [here](https://github.com/instana/python-sensor/blob/fix_requirements/tests/conftest.py#L33-L39) to prevent running gcs tests on the runtime jobs.

**Review by GSVarsha — APPROVED** [review]
LGTM!

**Comment by CagriYonca** [comment]
I'm closing this PR. The PR below solves dependency problems with google-cloud package for Python 3.11 and 3.12:
https://github.com/instana/python-sensor/pull/667

**Comment by CagriYonca** [comment]
I'm closing this PR. The PR below solves dependency problems with google-cloud package for Python 3.11 and 3.12:
https://github.com/instana/python-sensor/pull/667

**Pull request closed without merging**